PocketKit/Sources/PocketKit/Settings/SettingsView.swift

```diff
@@ -0,0 +1,57 @@
+import SwiftUI
+import Textile
+
+
+class SettingsViewController: UIHostingController<SettingsView> {
+   override init(rootView: SettingsView) {
+        super.init(rootView: rootView)
+        
+        UITableView.appearance(whenContainedInInstancesOf: [Self.self]).backgroundColor = UIColor(.ui.grey7)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError()
+    }
+}
+
+struct SettingsView: View {
+    
+    @ObservedObject
+    var model: AccountViewModel
+
+    var body: some View {
+        VStack(spacing:0) {
+            Form {
+                Section(header: Text("Your Account").style(.settings.header)) {
+                    PremiumRow(status: .notSubscribed, destination: EmptyView())
+                    SettingsRowLink(title: "Reset Password", destination: EmptyView())
+                    SettingsRowLink(title: "Delete Account", destination: EmptyView())
+                }.textCase(nil)
```

Comment:
Awesome that these things are available, but let's go ahead and comment this bit out until we're ready to implement the rest of the premium workflow.